cognitedata/reveal
# fix(react-components): add localization fallback

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-993

## Description :pencil:
Removed dependency on `cdf-i18n-utils` and added translation component for localization support using JSON files 

https://github.com/cognitedata/reveal/assets/87521752/b1f95db8-cdbf-45e7-8953-4f7d6373bf9a


https://github.com/cognitedata/reveal/assets/87521752/a0072a4f-18cc-49ec-a682-2d0339382ba7


https://github.com/cognitedata/reveal/assets/87521752/b2eb470d-9fa4-4486-aed2-22ac31039e46


## How has this been tested? :mag:

In storybook, Data exploration, FDX & 3D-management

## Test instructions :information_source:

- `cd react-components && yarn && yarn build:watch:yalc`
- `yarn storybook`
- In storybook, run `Toolbar` example. In Browser, `Settings---->Perference----->Change Language to German`. Reload the page to check the `Help` screen for Germán language support
- In fusion repo, `yalc add @cognite/reveal-react-components && yarn && nx serve data-exploration/business-shell/3d-management`
- For Data exploration, select any `360 image` & check the `Details` Button
- For FDX, change loading RevealContainer by adding `appLanguage={getLanguage()}` prop in `ThreeDContent.tsx` file. And change the language in user profile to see the changes


## Checklist :ballot_box_with_check:
